### PR TITLE
Scroll behavior

### PR DESCRIPTION
**Pull request by AdnanOlarmmi**
## Update CSS files 📝 

- Made the scroll behavior smooth for better professionalism 👨‍🏫 😃 

### Before: 
before

### After: 
after

Thank you.

### CODE REVIEW

**Review by harish-sethuraman — COMMENTED** [review]
Would you like to add the same to beta site as well?

**Comment by AdnanOlarmmi** [comment]
Hi @harish-sethuraman, yes I will. Please give me about 3hours... I will add it asap 🙂

**Comment by AdnanOlarmmi** [comment]
Hi @harish-sethuraman, Done 😃

File: beta/src/styles/algolia.css [hunk 1]
```diff
@@ -19,6 +19,9 @@
   --fds-animation-fade-in: cubic-bezier(0, 0, 1, 1);
   --fds-animation-fade-out: cubic-bezier(0, 0, 1, 1);
 }
+html {
+  scroll-behavior: smooth;
+}
```
Comment (harish-sethuraman) [inline beta/src/styles/algolia.css:24]:
```suggestion
```
It works when we have it at one place alone 👀 we'd not need it inside almost all css files.

File: beta/src/styles/sandpack.css [hunk 2]
```diff
@@ -35,6 +35,10 @@
   --sp-font-lineHeight: 24px;
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
```
Comment (harish-sethuraman) [inline beta/src/styles/sandpack.css:41]:
```suggestion
```

File: src/css/algolia.css [hunk 3]
```diff
@@ -4,6 +4,10 @@
  * Customized Algolia search result styles.
  */
 
+html {
+  scroll-behavior: smooth;
+}
+
```
Comment (harish-sethuraman) [inline src/css/algolia.css:10]:
```suggestion
```

**Comment by AdnanOlarmmi** [comment]
@harish-sethuraman committed duplicate removal, thanks.

**Review by harish-sethuraman — APPROVED** [review]
LGTM. Thanks!

**Comment by AdnanOlarmmi** [comment]
@harish-sethuraman, you're welcome and thank you too 🙂👍

**Comment by AdnanOlarmmi** [comment]
Hi @gaearon 🙂, I understand that functionality should be considered over aesthetics, not that aesthetics should be ignored. This is just an artistic quality of a sensory experience, which doesn't even take a centisecond to traverse, not to the extent of "slowing down the user experience". Platforms like Mozilla Developer (MDN) use it, and scrolling abruptly is also not a good user experience. I suggest that it should still be implemented.

Thanks. 🙂

**Comment by gaearon** [comment]
It is definitely noticeably slower, and the thing I'm most worried about is the initialization of sandboxes "along the way" on the beta site that initializes them as they get into viewport. This is an unnecessary cost that the jump avoids.

**Pull request merged**